app/views/fsss_view.py

```diff
@@ -2,7 +2,12 @@
 from app.tabutils import tab_generation, table_generation, format_to_uk_dates
 from app.fieldmapping import map_employee_name
 
-
+def get_device_types(devices):
+    types_of_device = ""
+    if devices != []:
+        for device in devices:
+            types_of_device = types_of_device + str(device['Device ID']) + "\n"
```

Comment:
this is called types of device but it looks like it will just be a list of the device IDs, is this right?

Reply:
or does `device['Device ID']` return the whole device row?

Reply:
Yes this is wrong! I`ll update it to device['Device Type']